specs/EditExistingTemplateAndViewTemplate.spec

```diff
@@ -91,7 +91,7 @@ tags: 4600, Clicky Admin, 7862
 
 * Verify that templates "unused-template" are present
 * Verify can delete templates "unused-template"
-* Verify cannot add new template for template admin
+* Verify can add new template for group admin
```

Comment:
Shouldn't we have both the verifications? ie. `Verify can add new template for group admin` and `Verify cannot add new template for template admin`.